lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,81 @@
+.container {
+    background-color: #fff;
+    display: inline-flex;
+    flex-direction: column;
+
+    button,
+    svg {
+        color: #337ab7;
```

Comment:
Should use the color variable `$color-text-blue-dark` for this.

Reply:
This is now updated.